pkgs/development/libraries/tpm2-tss/default.nix

```diff
@@ -50,7 +54,7 @@ stdenv.mkDerivation rec {
   '';
 
   configureFlags = [
-    "--enable-unit"
+    (lib.optionalString (stdenv.buildPlatform == stdenv.hostPlatform) "--enable-unit")
     "--enable-integration"
```

Comment:
What does `--enable-integration` do? Assuming it runs integration tests, should they be run when cross-compiling?

I had this as a quickfix when I needed to build a pkgsStatic.tpm2-tss:
```
    tpm2-tss = prev.tpm2-tss.overrideAttrs (old: {
      configureFlags = [];
    });

Reply:
Gated that as well.